### ISSUE TITLE
Exception spam with Ping passthrough

### ISSUE THREAD

**Issue by Artuto** [GeyserMC/Geyser#1631]
**Describe the bug**
The console gets spammed constantly with this error:

```
java.net.SocketException: Connection reset
        at java.base/java.net.SocketInputStream.read(Unknown Source)
        at java.base/java.net.SocketInputStream.read(Unknown Source)
        at java.base/java.net.SocketInputStream.read(Unknown Source)
        at java.base/java.io.DataInputStream.readByte(Unknown Source)
        at com.nukkitx.nbt.util.VarInts.decodeUnsigned(VarInts.java:48)
        at com.nukkitx.nbt.util.VarInts.readUnsignedInt(VarInts.java:25)
        at org.geysermc.connector.ping.GeyserLegacyPingPassthrough.run(GeyserLegacyPingPassthrough.java:101)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.base/java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.base/java.lang.Thread.run(Unknown Source
```

**To Reproduce**
1. Enable Ping passthrough
2. Wait

**Expected behavior**
Not get the console spammed

**Screenshots / Videos**

**Server Version**
1.16.4 Waterfall - Standalone Geyser

**Geyser Version**
This server is running Geyser version 1.2.0-SNAPSHOT (git-master-2f294c9) (Java: 1.16.4, Bedrock: 1.16.100/1.16.101 - 1.16.200.56)

**Minecraft: Bedrock Edition Version**
Does not apply

**Additional Context**

**Comment by toinouH** [comment]
Hello, could you provide a dump please ?
`/geyser dump` in the console

**Comment by Artuto** [comment]
https://dump.geysermc.org/HEtvQOur841rFA33QviqkjrtSKBO0sDB

**Comment by RednedEpic** [comment]
Closed in favor of #2323.
